addons/lunch/static/src/scss/lunch_list.scss

```diff
@@ -1,11 +1,7 @@
 .o_lunch_content {
-    .o_list_button {
-        width: 1px;
-    }
-
     .o_lunch_list_view {
         td:last-child {
             padding-right: 16px;
```

Comment:
This should not be here :/
It's a workaround for an issue we have managing paddings in list-view.

Currently we have 2 design, one for standalone list-views and another one for embedded list-views (into form-view  or modals for example...).

You should fix it in form-view directly, [extending the selector](https://github.com/odoo/enterprise/blob/master/web_enterprise/static/src/legacy/scss/list_view.scss#L170) in order to match list view inside `o_controller_with_searchpanel`